pkg/nvml/return.go

```diff
@@ -20,17 +20,76 @@ import (
 
 // nvml.ErrorString()
 func (l *library) ErrorString(r Return) string {
-	return r.Error()
+	return r.String()
 }
```

Comment:
How does an end-user ever get ahold of a `Return` value to pass it to this function?